### DISCUSSION TITLE
EF migrations

### DISCUSSION THREAD

**Discussion by shifttechgs** [jasontaylordev/CleanArchitecture#1135] [Q&A] [answered]
please help, getting an error using donet8  after running

 dotnet ef migrations add "SampleMigration" --project src\Infrastructure --startup-project src\Web --output-dir Data\Migrations

Error:  error MSB4057: The target "GetEFProjectMetadata" d
oes not exist in the project.
Unable to retrieve project metadata. Ensure it's an SDK-style project. If you're using a custom BaseIntermediateOutputPath or MSBuildProjectExtensionsPath values, Use the --msbuildprojectextensionspath option.

**Comment by jasontaylordev** [comment, marked as answer]
See https://github.com/jasontaylordev/CleanArchitecture/issues/1137#issuecomment-1981871242
